src/sinol_make/util.py

```diff
@@ -108,6 +108,39 @@ def check(path):
 		return None
 
 
+def save_config(config):
+	"""
+	Function to save nicely formated config.yml
+	"""
+
+	order = [
+		"title",
+		"memory_limit",
+		"time_limit",
+		"override_limits",
+		"scores",
+		{
+			"key": "sinol_expected_scores",
+			"default_flow_style": None
+		}
+	]
+
+	config = config.copy()
+	with open("config.yml", "w") as config_file:
+		for field in order:
+			if isinstance(field, dict):
+				if field["key"] in config:
```

Comment:
Those `if`s are still not obvious to me. If you also think they are non-obvious, we can add comments explaining them.

Reply:
I can add some comments here

Reply:
3fb7b53d5b20c2558451734177394e50e8774d77